instance/models/instance.py

```diff
@@ -94,6 +96,8 @@ class Instance(ValidateModelMixin, TimeStampedModel):
     STOPPED = 'stopped'
     TERMINATING = 'terminating'
 
+    ERR_PROVISIONING_FAILED = 'provisioning_failed'
```

Comment:
I'm not quite sure `SUCCESS` is a useful value for `status_progress` – wouldn't successful conclusion of a status usually mean we are moving on to the next state?  I think we only need the states `RUNNING` and `FAILED`, so maybe we could make do with a Boolean flag.